### PR TITLE
SC FSM intermittent test failure

### PR DESCRIPTION
**Pull request by velzevur**
PT [Channel create Intermittent test failure](https://www.pivotaltracker.com/story/show/159304186)
PT description seems to be broken and is inconsistent with the title.

This PR fixes the intermittent test failure due to a race in the multiple channels' test.

### CODE REVIEW

**Comment by velzevur** [comment]
This is not in the scope or release 21 but rather 22.

Until then this PR is marked as a `WIP`.

**Comment by lucafavatella** [comment]
Seems in good shape for review&merge. Removing WIP label.

File: apps/aechannel/test/aesc_fsm_SUITE.erl [hunk 1]
```diff
@@ -1035,6 +1036,20 @@ mine_blocks(_, _, #{mine_blocks := false}) ->
 mine_blocks(Node, N, _) ->
     aecore_suite_utils:mine_key_blocks(aecore_suite_utils:node_name(Node), N).
 
+mine_all_transactions(_Node, MaxAttempts) when MaxAttempts =< 0 ->
```
Comment (hanssv) [inline apps/aechannel/test/aesc_fsm_SUITE.erl]: Noooo ;-)

Use `aecore_suite_utils:mine_blocks_until_txs_on_chain(Node, Txs, MaxAttempts)`

**Comment by velzevur** [comment]
tests failed with an unrelated
```

%%% aeu_requests_SUITE ==> application_test: FAILED
%%% aeu_requests_SUITE ==> {errors_in_lager_log,["08:34:04.327 [error] CRASH REPORT Process <0.15.1> with 0 neighbours exited with reason: {aborted,{no_exists,aec_chain_state}} in mnesia:wrap_trans/6 line 493\n\n",
                      "08:34:04.327 [error] Supervisor aecore_sup had child aec_tx_pool started with aec_tx_pool:start_link() at undefined exit with reason {aborted,{no_exists,aec_chain_state}} in context start_error\n\n",
                      "08:34:04.327 [error] CRASH REPORT Process <0.7.1> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,aec_tx_pool,{aborted,{no_exists,aec_chain_state}}}},{aecore_app,start,[normal,[]]}} in application_master:init/4 line 134\n\n"]}```
that had been removed in master. Restarted tests

**Pull request merged**